tests/phpunit/tests/query/stickies.php

```diff
@@ -6,7 +6,8 @@
  * @group query
  */
 class Tests_Query_Stickies extends WP_UnitTestCase {
-	public static $posts = array();
+	public static $posts         = array();
+	protected $sticky_query_vars = array();
```

Comment:
```suggestion
	public static $posts = array();
```

But only if you use the utility class.